tests/test_database.py

```diff
@@ -80,3 +79,33 @@ def test_current_timestamp(self):
         db = get_conn(self.db_cls)
         res = db.query(current_timestamp(), datetime)
         assert isinstance(res, datetime), (res, type(res))
+
+    def test_correct_timezone(self):
+        name = "tbl_" + random_table_suffix()
+        db = get_conn(self.db_cls)
+        tbl = table(db.parse_table_name(name), schema={
+            "id": int, "created_at": "timestamp_tz(9)", "updated_at": "timestamp_tz(9)"
```

Comment:
`timestamp_tz(9)` is database vendor agnostic, unfortunately. I have no idea how to make this work with the generic approach. Any hints on this? Thanks.